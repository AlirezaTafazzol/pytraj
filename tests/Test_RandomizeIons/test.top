%VERSION  VERSION_STAMP = V0001.000  DATE = 01/14/15  18:29:58                  
%FLAG TITLE                                                                     
%FORMAT(20a4)                                                                   
                                                                                
%FLAG POINTERS                                                                  
%FORMAT(10I8)                                                                   
      14      17       0       0       0       0       0       0       0       0
      14      14       0       0       0      37      68      49      27       1
       0       0       0       0       0       0       0       0       1       0
       0
%FLAG ATOM_NAME                                                                 
%FORMAT(20a4)                                                                   
P   P   P   P   P   P   P   P   P   P   P   P   P   P   
%FLAG CHARGE                                                                    
%FORMAT(5E16.8)                                                                 
  2.12508463E+01  2.12508463E+01  2.12508463E+01  2.12508463E+01  2.12508463E+01
  2.12508463E+01  2.12508463E+01  2.12508463E+01  2.12508463E+01  2.12508463E+01
  2.12508463E+01  2.12508463E+01  2.12508463E+01  2.12508463E+01
%FLAG ATOMIC_NUMBER                                                             
%FORMAT(10I8)                                                                   
      15      15      15      15      15      15      15      15      15      15
      15      15      15      15
%FLAG MASS                                                                      
%FORMAT(5E16.8)                                                                 
  3.09700000E+01  3.09700000E+01  3.09700000E+01  3.09700000E+01  3.09700000E+01
  3.09700000E+01  3.09700000E+01  3.09700000E+01  3.09700000E+01  3.09700000E+01
  3.09700000E+01  3.09700000E+01  3.09700000E+01  3.09700000E+01
%FLAG ATOM_TYPE_INDEX                                                           
%FORMAT(10I8)                                                                   
      12      12      12      12      12      12      12      12      12      12
      12      12      12      12
%FLAG NUMBER_EXCLUDED_ATOMS                                                     
%FORMAT(10I8)                                                                   
       1       1       1       1       1       1       1       1       1       1
       1       1       1       1
%FLAG NONBONDED_PARM_INDEX                                                      
%FORMAT(10I8)                                                                   
       1       2       4       7      11      16      22      29      37      46
      56      67      79      92     106     121     137       2       3       5
       8      12      17      23      30      38      47      57      68      80
      93     107     122     138       4       5       6       9      13      18
      24      31      39      48      58      69      81      94     108     123
     139       7       8       9      10      14      19      25      32      40
      49      59      70      82      95     109     124     140      11      12
      13      14      15      20      26      33      41      50      60      71
      83      96     110     125     141      16      17      18      19      20
      21      27      34      42      51      61      72      84      97     111
     126     142      22      23      24      25      26      27      28      35
      43      52      62      73      85      98     112     127     143      29
      30      31      32      33      34      35      36      44      53      63
      74      86      99     113     128     144      37      38      39      40
      41      42      43      44      45      54      64      75      87     100
     114     129     145      46      47      48      49      50      51      52
      53      54      55      65      76      88     101     115     130     146
      56      57      58      59      60      61      62      63      64      65
      66      77      89     102     116     131     147      67      68      69
      70      71      72      73      74      75      76      77      78      90
     103     117     132     148      79      80      81      82      83      84
      85      86      87      88      89      90      91     104     118     133
     149      92      93      94      95      96      97      98      99     100
     101     102     103     104     105     119     134     150     106     107
     108     109     110     111     112     113     114     115     116     117
     118     119     120     135     151     121     122     123     124     125
     126     127     128     129     130     131     132     133     134     135
     136      -1     137     138     139     140     141     142     143     144
     145     146     147     148     149     150     151      -1     153
%FLAG RESIDUE_LABEL                                                             
%FORMAT(20a4)                                                                   
G   G   C   G   C   C   C3  G   G   C   G   C   C   C3  
%FLAG RESIDUE_POINTER                                                           
%FORMAT(10I8)                                                                   
       1       2       3       4       5       6       7       8       9      10
      11      12      13      14
%FLAG BOND_FORCE_CONSTANT                                                       
%FORMAT(5E16.8)                                                                 
  2.30000000E+02  5.53000000E+02  3.40000000E+02  3.20000000E+02  3.10000000E+02
  3.20000000E+02  4.61000000E+02  4.34000000E+02  4.81000000E+02  4.83000000E+02
  4.34000000E+02  4.27000000E+02  5.70000000E+02  4.18000000E+02  4.47000000E+02
  5.20000000E+02  4.14000000E+02  3.67000000E+02  5.29000000E+02  4.40000000E+02
  4.36000000E+02  3.40000000E+02  3.37000000E+02  3.40000000E+02  3.10000000E+02
  3.20000000E+02  3.20000000E+02  5.25000000E+02  4.57000000E+02  3.67000000E+02
  4.27000000E+02  3.67000000E+02  5.49000000E+02  4.48000000E+02  4.24000000E+02
  5.53000000E+02  5.53000000E+02
%FLAG BOND_EQUIL_VALUE                                                          
%FORMAT(5E16.8)                                                                 
  1.61000000E+00  9.60000000E-01  1.09000000E+00  1.41000000E+00  1.52600000E+00
  1.41000000E+00  1.35400000E+00  1.01000000E+00  1.34000000E+00  1.33900000E+00
  1.01000000E+00  1.38100000E+00  1.22900000E+00  1.38800000E+00  1.41900000E+00
  1.37000000E+00  1.39100000E+00  1.08000000E+00  1.30400000E+00  1.37100000E+00
  1.37400000E+00  1.09000000E+00  1.47500000E+00  1.09000000E+00  1.52600000E+00
  1.41000000E+00  1.41000000E+00  1.48000000E+00  1.35800000E+00  1.08000000E+00
  1.43300000E+00  1.08000000E+00  1.35000000E+00  1.36500000E+00  1.38300000E+00
  9.57200000E-01  1.51360000E+00
%FLAG ANGLE_FORCE_CONSTANT                                                      
%FORMAT(5E16.8)                                                                 
  1.00000000E+02  4.50000000E+01  5.00000000E+01  5.00000000E+01  5.50000000E+01
  5.00000000E+01  5.00000000E+01  5.00000000E+01  1.00000000E+02  3.50000000E+01
  7.00000000E+01  5.00000000E+01  7.00000000E+01  5.00000000E+01  7.00000000E+01
  7.00000000E+01  8.00000000E+01  6.30000000E+01  5.00000000E+01  7.00000000E+01
  8.00000000E+01  7.00000000E+01  7.00000000E+01  7.00000000E+01  7.00000000E+01
  5.00000000E+01  7.00000000E+01  7.00000000E+01  5.00000000E+01  5.00000000E+01
  7.00000000E+01  7.00000000E+01  7.00000000E+01  5.00000000E+01  5.00000000E+01
  7.00000000E+01  7.00000000E+01  4.00000000E+01  5.00000000E+01  5.00000000E+01
  6.00000000E+01  5.00000000E+01  3.50000000E+01  5.00000000E+01  5.00000000E+01
  4.00000000E+01  5.00000000E+01  5.00000000E+01  5.50000000E+01  5.00000000E+01
  5.00000000E+01  1.40000000E+02  1.00000000E+02  8.00000000E+01  7.00000000E+01
  5.00000000E+01  7.00000000E+01  7.00000000E+01  5.00000000E+01  7.00000000E+01
  5.00000000E+01  6.30000000E+01  5.00000000E+01  7.00000000E+01  7.00000000E+01
  8.00000000E+01  7.00000000E+01  7.00000000E+01
%FLAG ANGLE_EQUIL_VALUE                                                         
%FORMAT(5E16.8)                                                                 
  1.88897066E+00  1.79070858E+00  1.91113635E+00  1.91113635E+00  1.89368305E+00
  1.91113635E+00  1.91113635E+00  1.91113635E+00  2.10312265E+00  2.09439600E+00
  2.08217869E+00  2.09439600E+00  1.95826026E+00  2.05948940E+00  2.02458280E+00
  2.15199189E+00  2.10486798E+00  2.08043336E+00  2.03854544E+00  2.18515316E+00
  2.24798504E+00  1.94255229E+00  2.22878641E+00  2.26892900E+00  1.92684432E+00
  2.14762857E+00  1.83957782E+00  1.81165254E+00  1.91113635E+00  2.14762857E+00
  1.98793087E+00  1.85354046E+00  2.20260646E+00  1.91113635E+00  1.91113635E+00
  2.24798504E+00  2.19562514E+00  1.91113635E+00  1.91113635E+00  1.91113635E+00
  1.91113635E+00  1.91113635E+00  1.91113635E+00  1.91113635E+00  1.91113635E+00
  1.91113635E+00  1.91113635E+00  1.91113635E+00  1.89368305E+00  1.91113635E+00
  1.91113635E+00  2.09265067E+00  2.10312265E+00  2.13802925E+00  2.10312265E+00
  2.15199189E+00  2.09614133E+00  2.12057595E+00  2.08916001E+00  2.12232128E+00
  2.08916001E+00  2.04203610E+00  2.07868803E+00  2.11533996E+00  2.06996138E+00
  2.11010397E+00  2.11533996E+00  2.05250808E+00
%FLAG DIHEDRAL_FORCE_CONSTANT                                                   
%FORMAT(5E16.8)                                                                 
  1.85181000E-01  1.25653100E+00  3.54858000E-01  1.17500000E+00  1.44000000E-01
  2.50000000E-01  0.00000000E+00  1.66666667E-01  3.83333333E-01  1.55555556E-01
  1.60000000E-01  2.50000000E-01  1.20000000E+00  0.00000000E+00  2.40000000E+00
  4.80000000E+00  1.50000000E+00  3.00000000E+00  1.35000000E+00  5.45000000E+00
  4.15000000E+00  1.70000000E+00  1.00000000E+01  1.65000000E+00  2.55000000E+00
  1.00000000E-01  3.83000000E-01  7.05100000E-01  1.06546000E+00  4.42730000E-01
  2.56020000E-01  6.50000000E-01  2.00000000E-01  2.50000000E-01  1.80000000E-01
  1.17804000E+00  9.21020000E-02  9.62830000E-01  4.00000000E+00  1.85000000E+00
  6.65000000E+00  1.45000000E+00  1.22506000E+00  1.63459000E+00  9.37470000E-01
  3.10330000E-01  1.00000000E+00  1.10000000E+00  1.05000000E+01
%FLAG DIHEDRAL_PERIODICITY                                                      
%FORMAT(5E16.8)                                                                 
  1.00000000E+00  2.00000000E+00  3.00000000E+00  2.00000000E+00  3.00000000E+00
  1.00000000E+00  3.00000000E+00  3.00000000E+00  3.00000000E+00  3.00000000E+00
  3.00000000E+00  3.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  3.00000000E+00  1.00000000E+00  2.00000000E+00  3.00000000E+00
  4.00000000E+00  2.00000000E+00  1.00000000E+00  2.00000000E+00  3.00000000E+00
  1.00000000E+00  2.00000000E+00  3.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  1.00000000E+00  2.00000000E+00  3.00000000E+00
  4.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
%FLAG DIHEDRAL_PHASE                                                            
%FORMAT(5E16.8)                                                                 
  5.54929070E-01  6.14285649E+00  6.23514744E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  3.14159400E+00
  3.14159400E+00  3.14159400E+00  3.14159400E+00  3.14159400E+00  3.14159400E+00
  3.14159400E+00  3.14159400E+00  3.14159400E+00  3.14159400E+00  3.14159400E+00
  3.14159400E+00  0.00000000E+00  1.30473540E+00  1.08709624E-01  2.94350428E+00
  6.93698862E-02  0.00000000E+00  3.14159400E+00  3.14159400E+00  0.00000000E+00
  3.33317067E+00  5.15976777E+00  6.07541257E+00  3.14159400E+00  3.14159400E+00
  3.14159400E+00  3.14159400E+00  2.56544660E+00  2.87571043E-01  3.24417403E+00
  5.61280675E-01  3.14159400E+00  3.14159400E+00  3.14159400E+00
%FLAG SCEE_SCALE_FACTOR                                                         
%FORMAT(5E16.8)                                                                 
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00  1.20000000E+00
  1.20000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
%FLAG SCNB_SCALE_FACTOR                                                         
%FORMAT(5E16.8)                                                                 
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00  2.00000000E+00
  2.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
%FLAG SOLTY                                                                     
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00
%FLAG LENNARD_JONES_ACOEF                                                       
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  5.81803229E+05  0.00000000E+00  7.91544157E+05
  1.04308023E+06  0.00000000E+00  4.66922514E+04  6.78771368E+04  3.25969625E+03
  0.00000000E+00  4.58874091E+05  6.28541240E+05  3.63097246E+04  3.61397723E+05
  0.00000000E+00  3.15360051E+04  4.68930885E+04  2.09814978E+03  2.44050579E+04
  1.32801250E+03  0.00000000E+00  7.44975864E+05  9.95480466E+05  6.20665997E+04
  5.89818288E+05  4.24594555E+04  9.44293233E+05  0.00000000E+00  7.01803794E+05
  9.24822270E+05  6.01816484E+04  5.57281136E+05  4.15766412E+04  8.82619071E+05
  8.19971662E+05  0.00000000E+00  4.09427581E+04  5.98885646E+04  2.82099197E+03
  3.17965283E+04  1.80748014E+03  5.46147253E+04  5.30987710E+04  2.43828624E+03
  0.00000000E+00  4.71003287E+05  6.47841731E+05  3.69471530E+04  3.70622491E+05
  2.47595516E+04  6.06829342E+05  5.74393458E+05  3.23283631E+04  3.79876399E+05
  0.00000000E+00  1.40467023E+03  2.56678134E+03  5.94667300E+01  1.03954408E+03
  3.20009193E+01  2.12601181E+03  2.27577561E+03  4.90239937E+01  1.02595236E+03
  1.39982777E-01  0.00000000E+00  1.98683736E+06  2.54188684E+06  1.81087383E+05
  1.58759528E+06  1.27716552E+05  2.45746558E+06  2.25370349E+06  1.60682053E+05
  1.64263766E+06  8.41065839E+03  6.02589390E+06  0.00000000E+00  4.96707306E+04
  7.18621074E+04  3.50301067E+03  3.86654993E+04  2.26273716E+03  6.58473870E+04
  6.37148278E+04  3.03448006E+03  3.93690817E+04  6.63368273E+01  1.90880272E+05
  3.76169105E+03  0.00000000E+00  6.00750218E+04  8.59947003E+04  4.33325458E+03
  4.68711055E+04  2.82099197E+03  7.91627154E+04  7.62451550E+04  3.76169105E+03
  4.77908183E+04  8.90987508E+01  2.26202022E+05  4.64559155E+03  5.71629601E+03
  0.00000000E+00  1.02772745E+05  1.49995867E+05  7.11465296E+03  7.98522793E+04
  4.56604314E+03  1.36919347E+05  1.32990267E+05  6.15221292E+03  8.12116828E+04
  1.25820601E+02  4.01624254E+05  7.65043029E+03  9.47658423E+03  1.55205818E+04
  0.00000000E+00  5.82511344E+05  7.85890042E+05  4.75728872E+04  4.60252016E+05
  3.23234673E+04  7.42364908E+05  6.96790708E+05  4.17832254E+04  4.72934643E+05
  1.52094041E+03  1.95739541E+06  5.05435913E+04  6.09592309E+04  1.04820861E+05
  5.81935564E+05  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00
%FLAG LENNARD_JONES_BCOEF                                                       
%FORMAT(5E16.8)                                                                 
  0.00000000E+00  0.00000000E+00  6.99746810E+02  0.00000000E+00  6.93079947E+02
  6.75612247E+02  0.00000000E+00  1.03606917E+02  1.06076943E+02  1.43076527E+01
  0.00000000E+00  5.89183300E+02  5.85549272E+02  8.66220817E+01  4.95732238E+02
  0.00000000E+00  8.51470647E+01  8.81685417E+01  1.14788417E+01  7.10161395E+01
  9.13231543E+00  0.00000000E+00  7.50714425E+02  7.36907417E+02  1.13252061E+02
  6.33305958E+02  9.36708259E+01  8.01323529E+02  0.00000000E+00  6.14502845E+02
  5.99015525E+02  9.40505980E+01  5.19163331E+02  7.81725398E+01  6.53361429E+02
  5.31102864E+02  0.00000000E+00  9.59185913E+01  9.85097219E+01  1.31591746E+01
  8.01410250E+01  1.05333009E+01  1.05031585E+02  8.73413012E+01  1.20953369E+01
  0.00000000E+00  6.29300710E+02  6.26720080E+02  9.21192136E+01  5.29252520E+02
  7.54103785E+01  6.77220874E+02  5.55666448E+02  8.51921330E+01  5.64885984E+02
  0.00000000E+00  1.79702257E+01  2.06278363E+01  1.93248820E+00  1.46567808E+01
  1.41762397E+00  2.09604198E+01  1.82891803E+01  1.73473071E+00  1.53505284E+01
  9.37598976E-02  0.00000000E+00  1.27682121E+03  1.22636552E+03  2.01468199E+02
  1.08210555E+03  1.69194406E+02  1.34630496E+03  1.08732781E+03  1.87626421E+02
  1.16041466E+03  4.34187588E+01  2.19561270E+03  0.00000000E+00  1.05648788E+02
  1.07908863E+02  1.46638650E+01  8.83744760E+01  1.17854122E+01  1.15327881E+02
  9.56748258E+01  1.34932874E+01  9.40124296E+01  2.01792524E+00  2.04498832E+02
  1.50233639E+01  0.00000000E+00  1.16187983E+02  1.18043746E+02  1.63092814E+01
  9.73010751E+01  1.31591746E+01  1.26451907E+02  1.04660679E+02  1.50233639E+01
  1.03580945E+02  2.33864085E+00  2.22617115E+02  1.66953734E+01  1.85196588E+01
  0.00000000E+00  2.36133213E+02  2.42242668E+02  3.24719551E+01  1.97338832E+02
  2.60136673E+01  2.58405233E+02  2.14778698E+02  2.98534933E+01  2.09807371E+02
  4.31824675E+00  4.60918488E+02  3.32906603E+01  3.70514656E+01  7.36779156E+01
  0.00000000E+00  6.45512297E+02  6.36687196E+02  9.64152120E+01  5.44002597E+02
  7.94739459E+01  6.90894667E+02  5.64503554E+02  8.93335697E+01  5.81361517E+02
  1.72393904E+01  1.16838958E+03  9.82532117E+01  1.07902982E+02  2.19857570E+02
  5.94825035E+02  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00  0.00000000E+00
  0.00000000E+00  0.00000000E+00  0.00000000E+00
%FLAG BONDS_INC_HYDROGEN                                                        
%FORMAT(10I8)                                                                   

%FLAG BONDS_WITHOUT_HYDROGEN                                                    
%FORMAT(10I8)                                                                   

%FLAG ANGLES_INC_HYDROGEN                                                       
%FORMAT(10I8)                                                                   

%FLAG ANGLES_WITHOUT_HYDROGEN                                                   
%FORMAT(10I8)                                                                   

%FLAG DIHEDRALS_INC_HYDROGEN                                                    
%FORMAT(10I8)                                                                   

%FLAG DIHEDRALS_WITHOUT_HYDROGEN                                                
%FORMAT(10I8)                                                                   

%FLAG EXCLUDED_ATOMS_LIST                                                       
%FORMAT(10I8)                                                                   
       0       0       0       0       0       0       0       0       0       0
       0       0       0       0
%FLAG HBOND_ACOEF                                                               
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG HBOND_BCOEF                                                               
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG HBCUT                                                                     
%FORMAT(5E16.8)                                                                 
  0.00000000E+00
%FLAG AMBER_ATOM_TYPE                                                           
%FORMAT(20a4)                                                                   
P   P   P   P   P   P   P   P   P   P   P   P   P   P   
%FLAG TREE_CHAIN_CLASSIFICATION                                                 
%FORMAT(20a4)                                                                   
M   M   M   M   M   M   M   M   M   M   M   M   M   M   
%FLAG JOIN_ARRAY                                                                
%FORMAT(10I8)                                                                   
       0       0       0       0       0       0       0       0       0       0
       0       0       0       0
%FLAG IROTAT                                                                    
%FORMAT(10I8)                                                                   
       0       0       0       0       0       0       0       0       0       0
       0       0       0       0
%FLAG RADIUS_SET                                                                
%FORMAT(1a80)                                                                   
modified Bondi radii (mbondi)                                                   
%FLAG RADII                                                                     
%FORMAT(5E16.8)                                                                 
  1.85000000E+00  1.85000000E+00  1.85000000E+00  1.85000000E+00  1.85000000E+00
  1.85000000E+00  1.85000000E+00  1.85000000E+00  1.85000000E+00  1.85000000E+00
  1.85000000E+00  1.85000000E+00  1.85000000E+00  1.85000000E+00
%FLAG SCREEN                                                                    
%FORMAT(5E16.8)                                                                 
  8.60000000E-01  8.60000000E-01  8.60000000E-01  8.60000000E-01  8.60000000E-01
  8.60000000E-01  8.60000000E-01  8.60000000E-01  8.60000000E-01  8.60000000E-01
  8.60000000E-01  8.60000000E-01  8.60000000E-01  8.60000000E-01
