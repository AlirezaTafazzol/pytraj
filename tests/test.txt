None
Para
Anti
3-10
Alpha
Pi
Turn
Bend

