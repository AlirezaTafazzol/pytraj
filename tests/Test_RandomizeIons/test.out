['../../bin/cpptraj', 'adh206.ff10.tip3p.parm7.gz', 'ptraj.in']

CPPTRAJ: Trajectory Analysis. V15.22b
    ___  ___  ___  ___
     | \/ | \/ | \/ | 
    _|_/\_|_/\_|_/\_|_

| Date/time: 01/09/15  02:14:35
| Available memory: 762.969 MB

INPUT: Reading Input from file ptraj.in
  [trajin adh206.tip3p.rst7.gz]
  [randomizeions @Na+ around :1-16 by 5.0 overlap 3.0]
    RANDOMIZEIONS: swapping the postions of the ions in mask [@Na+]
                   with the solvent. No ions can get closer than 3.00 angstroms
                   to another ion.
                   No ion can get closer than 5.00 angstroms to mask [:1-16]
  [trajout random.crd title "Test"]
	Writing 'random.crd' as Amber Trajectory
  0: [randomizeions @Na+ around :1-16 by 5.0 overlap 3.0]
	Ion mask is [@Na+] (14 atoms)
	Around mask is [:1-16] (518 atoms)
100% Complete.

TIME: Total execution time: 0.3410 seconds.
--------------------------------------------------------------------------------
To cite CPPTRAJ use:
Daniel R. Roe and Thomas E. Cheatham, III, "PTRAJ and CPPTRAJ: Software for
  Processing and Analysis of Molecular Dynamics Trajectory Data". J. Chem.
  Theory Comput., 2013, 9 (7), pp 3084-3095.

